README.md

```diff
@@ -107,3 +111,96 @@ class Example extends Component {
 
 export default Example;
 ```
+
+## Example with custom pull to refresh for Android
+
+```javascript
+import React, { Component } from "react";
+import { View, TouchableOpacity, Text, RefreshControl, ActivityIndicator, Platform } from "react-native";
+import DraggableFlatList from "react-native-draggable-flatlist";
+
+const exampleData = [...Array(20)].map((d, index) => ({
+  key: `item-${index}`, // For example only -- don't use index as your key!
+  label: index,
+  backgroundColor: `rgb(${Math.floor(Math.random() * 255)}, ${index *
+    5}, ${132})`
+}));
+
+class Example extends Component {
+  state = {
+    data: exampleData,
+    refreshing: false,
+  };
+
+  renderItem = ({ item, index, drag, isActive }) => {
+    return (
+      <TouchableOpacity
+        style={{
+          height: 100,
+          backgroundColor: isActive ? "blue" : item.backgroundColor,
+          alignItems: "center",
+          justifyContent: "center"
+        }}
+        onLongPress={drag}
+      >
+        <Text
+          style={{
+            fontWeight: "bold",
+            color: "white",
+            fontSize: 32
+          }}
+        >
+          {item.label}
+        </Text>
+      </TouchableOpacity>
+    );
+  };
+
+  onRefresh: () => {
+    //update something
+  }
+
+  render() {
+
+    const iOSrefreshControl = (<RefreshControl onRefresh={this.onRefresh} refreshing={this.state.refreshing}  />)
+
+    const androidRefreshControl = (
+        <View
+              style={{
+                  height: 50,
+                  zIndex: -1,
+                  position: 'absolute',
+                  left: 0,
+                  right: 0,
+                  top: 0,
+              }}
+            >
+              <View style={{
+                  flex: 1,
+                  justifyContent: 'center',
+                  alignItems: 'center'
+              }}>
+                <ActivityIndicator size="large" />
+              </View>
+            </View>
+    )
+
+    return (
+      <View style={{ flex: 1 }}>
+        <DraggableFlatList
+          data={this.state.data}
+          renderItem={this.renderItem}
+          keyExtractor={(item, index) => `draggable-item-${item.key}`}
+          onDragEnd={({ data }) => this.setState({ data })}
+          refreshControl={Platform.OS === 'ios' ? iOSrefreshControl : androidRefreshControl}
+          onRefresh={this.onRefresh}
+          refreshing={this.state.refreshing}
```

Comment:
Why not access these two props from the same RefreshControl object you provide for iOS?

Reply:
> Why not access these two props from the same RefreshControl object you provide for iOS?

dude, it was 1 year ago. I dont remember anything about this now.